homeassistant/components/totalconnect/__init__.py

```diff
@@ -1 +1,61 @@
 """The totalconnect component."""
+import logging
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers import discovery
+from homeassistant.const import (CONF_PASSWORD, CONF_USERNAME)
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = 'Total Connect'
+
+DOMAIN = 'totalconnect'
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Required(CONF_USERNAME): cv.string,
+        vol.Required(CONF_PASSWORD): cv.string,
+    }),
+}, extra=vol.ALLOW_EXTRA)
+
+TOTALCONNECT_PLATFORMS = ['alarm_control_panel']
+
+
+class TotalConnectSystem:
```

Comment:
Please move this below `setup` function. We like to have the config schema and the setup function close.